var/spack/repos/builtin/packages/libtiff/package.py

```diff
@@ -6,7 +6,7 @@
 from spack.package import *
 
 
-class Libtiff(AutotoolsPackage):
+class Libtiff(CMakePackage):
```

Comment:
This should be changed to use Spack's new multi-build system support. See https://spack.readthedocs.io/en/latest/packaging_guide.html#multiple-build-systems for details.

Reply:
@adamjstewart I was unaware of this until about five minute ago. I've added a note to the top of this PR. I'll be refactoring as much of this PR as possible to use the multiple build systems. Feel free to hold of on review until that point, or review the in-situ version, whatever works.

Reply:
Yep, this was merged just yesterday. Should make Windows support significantly easier since packages can subclass AutotoolsPackage only on Linux/macOS and use a different build system on Windows. Hopefully we can refactor most packages like this.

Reply:
Personally I'm very excited about this PR. As you can see a number of these ports ended up fairly messy. I'll be refactoring as much as possible as I go.

Reply:
Same, packages like Python can also be refactored as well.

Reply:
I've refactored Python and Perl  and included them here.